src/app/child-dev-project/notes/notes-manager/notes-manager.component.ts

```diff
@@ -202,4 +222,16 @@ export class NotesManagerComponent implements OnInit, AfterViewInit {
   showDetails(entity: Note) {
     return this.formDialog.openDialog(NoteDetailsComponent, entity);
   }
+
+  public getColor(entity: Note): string {
```

Comment:
the same method is required in the child-notes-section, that's why it was move to the common place - the note entity class - to be reused easily